lookout/style/typos/analyzer.py

```diff
@@ -26,6 +26,7 @@
     ("line_number", int),                                     # line number for the comment
     ("identifier", str),                                      # identifier where typo is found
     ("candidates", Iterable[Candidate]),                      # suggested identifiers
+    ("identifiers_number", int),                              # Number of all analyzed identifiers
```

Comment:
Do you mean all inside a file? Then we should do it differently because TypoFix must correspond to a single id fix

Reply:
Great idea!

Reply:
ok, I add a todo for now. I'll address it next week